locales/en_US.json

```diff
@@ -154,6 +162,24 @@
 				"caption": "Web and server development",
 				"picture": "https://github.com/hauntii.png",
 				"github": "https://github.com/hauntii"
+			},
+			{
+				"name": "MatthewL246",
+				"caption": "Server architecture and community work",
```

Comment:
Personally, I'd say my role is closer to "DevOps" than "server architecture."

Reply:
I put anything related to docker as server architecture. I dunno if we want to be more specific for userfacing docs like this

Reply:
(Mario has "Systems research and server architecture"; Billy has "Preservationist and server architecture" - I think Billy's contributions revolve around CloudFlare usage and configuration so it's kinda vague)

Reply:
Billy's main contributions, historically, have revolved around preserving packet dumps. He's the main one who was getting us information at a large scale prior to our own mass collection. The CloudFlare stuff came later, and is now mostly what he does since packet dumps are kinda dead